providers/os/id/networki/linux_routes.go

```diff
@@ -0,0 +1,313 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+//go:build linux
```

Comment:
this should not be a build tag since it will fail when you ssh from macos to a linux box